bin/submit.sh

```diff
@@ -34,9 +34,9 @@ fi
 if [[ $CHUNJUN_HOME && -z $CHUNJUN_HOME ]];then
     export CHUNJUN_HOME=$CHUNJUN_HOME
 else
-    export CHUNJUN_HOME="$(cd "`dirname "$0"`"/../chunjun-dist; pwd)"
+    export CHUNJUN_HOME="$(cd "`dirname "$0"`"/..; pwd)"
```

Comment:
yeah, your pr is considered the situation that CHUNJUN_HOME is chunjun-dist, like
```
ls chunjun-dist           
bin                     connector               dirty-data-collector    formats                 metrics
chunjun-core-master.jar ddl                     docker-build            lib                     restore-plugins
```
which means that user use chunjun from release package.
But we also need to consider the situation that CHUNJUN_HOME is chunjun(project path, parent path of chunjun-dist), like
```
 ls chunjun
CONTRIBUTING.md                 chunjun-assembly                chunjun-dist                    chunjun-restore                 flinkx-core                     lib
LICENSE                         chunjun-clients                 chunjun-docker                  chunjun-sql                     flinkx-dirtydata-collectors     mvnw
README.md                       chunjun-connectors              chunjun-examples                chunjun.iml                     flinkx-docker                   mvnw.cmd
README_CH.md                    chunjun-core                    chunjun-formats                 docs_zh                         flinkx-formats                  pom.xml
bin                             chunjun-dev                     chunjun-local-test              flinkx-clients                  flinkx-metrics                  website
build                           chunjun-dirty                   chunjun-metrics                 flinkx-connectors               flinkx-restore

```
which means that user use chunjun from Git clone.
So you need to consider this two situations using 'if' statement.